test/sweet_xml_test.exs

```diff
@@ -310,7 +310,7 @@ defmodule SweetXmlTest do
     result = readme |> xpath(~x"//matchup/name/text()") # `x` marks sigil for (x)path
     assert result == 'Match One'
 
-    # get the xml record of the name fo the first match
+    # get the xml record of the name for the first match
```

Comment:
Could you change this to `of` instead of `for` ? This would match with the examples section in the readme: https://github.com/kbrw/sweet_xml#examples